worlds/seaofthieves/Rules.py

```diff
@@ -0,0 +1,22 @@
+import random
+
+from worlds.seaofthieves.Items.Items import Items
+
+from .Configurations import SotOptionsDerived
+from BaseClasses import MultiWorld
+from .Regions.Regions import Regions, RegionAdder
+from .Regions.RegionConnectionRules import create_rules
+
+
+def set_rules(world: MultiWorld, options: SotOptionsDerived.SotOptionsDerived, player: int, regionAdder: RegionAdder):
```

Comment:
```suggestion
def set_rules(multiworld: MultiWorld, options: SotOptionsDerived.SotOptionsDerived, player: int, regionAdder: RegionAdder):
```
Definitely do not use `world: MultiWorld`
It is very confusing for reviewers and people referencing your code